File: GO/Cap04/c04ex10.go
// c04ex10.go

package main

import "fmt"

func main() {
  var A, B, X, C int

  fmt.Print("Entre valor para a variavel <A>: ")
  fmt.Scan(&A)

  fmt.Print("Entre valor para a variavel <B>: ")
  fmt.Scan(&B)

  fmt.Print("Entre valor para a variavel <X>: ")
  fmt.Scan(&X)

  if (!(X > 5)) {
    C = A + B
  } else {
    C = A - B
  }

  fmt.Println()

  fmt.Println("Resultado =", C)

  fmt.Println()
  fmt.Print("Tecle <Enter> para encerrar... ")
  fmt.Scanln()
}

File: RUBY/Cap04/c04ex10.rb
# c04ex10.rb

#!/usr/bin/env ruby

print "Entre valor para a variavel <A>: "
a = gets.chomp.to_i

print "Entre valor para a variavel <B>: "
b = gets.chomp.to_i

print "Entre valor para a variavel <X>: "
x = gets.chomp.to_i

if (!(x > 5))
  c = a + b
else
  c = a - b
end

puts
puts "Resultado = #{c}"

puts
print "Tecle <Enter> para encerrar... "
gets
